config/v1/types_authentication.go

```diff
@@ -43,13 +43,15 @@ type AuthenticationSpec struct {
 	// +optional
 	OAuthMetadata ConfigMapNameReference `json:"oauthMetadata"`
 
-	// webhookTokenAuthenticators configures remote token reviewers.
+	// webhookTokenAuthenticators is DEPRECATED, setting it has no effect.
+	WebhookTokenAuthenticators []WebhookTokenAuthenticator `json:"webhookTokenAuthenticators,omitempty"`
+
+	// webhookTokenAuthenticator configures a remote token reviewer.
 	// These remote authentication webhooks can be used to verify bearer tokens
-	// via the tokenreviews.authentication.k8s.io REST API.  This is required to
+	// via the tokenreviews.authentication.k8s.io REST API. This is required to
 	// honor bearer tokens that are provisioned by an external authentication service.
-	// The namespace for these secrets is openshift-config.
 	// +optional
-	WebhookTokenAuthenticators []WebhookTokenAuthenticator `json:"webhookTokenAuthenticators,omitempty"`
+	WebhookTokenAuthenticator *WebhookTokenAuthenticator `json:"webhookTokenAuthenticator,omitempty"`
```

Comment:
as written in Slack: you have to use an alternative type here because you make it stricter via validation. I would just make a copy and call one `DeprecatedWebhookTokenAuthenticator` and the other one here `WebhookTokenAuthenticator`.